File: SegurosAPI/src/main/java/com/clinica/api/seguros_service/repository/UsuarioRepository.java
package com.clinica.api.seguros_service.repository;

import com.clinica.api.seguros_service.model.Usuario;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UsuarioRepository extends JpaRepository<Usuario, Long> {
}

File: usuario-service/src/main/kotlin/com/clinica/api/usuario_service/repository/UsuarioRepository.kt
package com.clinica.api.usuario_service.repository

import com.clinica.api.usuario_service.model.Usuario
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.Optional

@Repository
interface UsuarioRepository : JpaRepository<Usuario, Long> {
    fun findByCorreo(correo: String): Optional<Usuario>
    fun existsByCorreo(correo: String): Boolean
}
